docs/reference/argocd.md

```diff
@@ -821,6 +821,7 @@ LogLevel | info | The log level to be used by the ArgoCD Repo Server. Valid opti
 LogFormat | text | The log format to be used by the ArgoCD Repo Server. Valid options are text or json.
 ExecTimeout | 180 | Execution timeout in seconds for rendering tools (e.g. Helm, Kustomize)
 Env | [Empty] | Environment to set for the repository server workloads
+Replicas | 1 | The number of replicas for the ArgoCD Repo Server. 
```

Comment:
Default to nil.  If this value set, it must be >= 0 This value is passed to replicas in Deployment spec.